### PR TITLE
Populating content

### PR DESCRIPTION
**Pull request by jakearchibald**

### CODE REVIEW

File: src/_includes/footer/index.njk [hunk 1]
```diff
@@ -7,18 +7,13 @@
   <div class="{% className css, 'divider' %}"></div>
   <ul class="{% className css, 'social' %}">
     <li><a href="https://twitter.com/chromiumdev" rel="noopener noreferrer" target="_blank"><span class="{% className css, 'icon-mask' %}" style="--icon: url(confboxAsset(/_includes/footer/assets/twitter.svg))">Twitter</span></a></li>
-    <li><a href="https://youtube.com/chromedevelopers" rel="noopener noreferrer" target="_blank"><span class="{% className css, 'icon-mask' %}" style="--icon: url(confboxAsset(/_includes/footer/assets/youtube.svg))">YouTube</span></a></li>
+    <li><a href="https://www.youtube.com/user/ChromeDevelopers" rel="noopener noreferrer" target="_blank"><span class="{% className css, 'icon-mask' %}" style="--icon: url(confboxAsset(/_includes/footer/assets/youtube.svg))">YouTube</span></a></li>
```
Comment (surma) [inline src/_includes/footer/index.njk]: The old link works as well and is what we show in our channel trailer 🤷‍♂

Comment (jakearchibald) [reply src/_includes/footer/index.njk]: I was just skipping the redirect, but feel free to revert

Comment (surma) [reply src/_includes/footer/index.njk]: Fair. I was going for pretty URLs :D

Comment (surma) [reply src/_includes/footer/index.njk]: (too lazy to revert, tbh)

**Comment by jakearchibald** [comment]
Ta, sorry for the uber PR. I just kinda kept coding until I got tired.

**Pull request merged**